### PR TITLE
Advanced search facets

### PR DESCRIPTION
**Pull request by LtChae**

### CODE REVIEW

File: apps/discovery_api/config/integration.exs [hunk 1]
```diff
@@ -49,6 +49,18 @@ config :discovery_api, :elasticsearch,
               type: "keyword",
               index: true
             },
+            keywordFacets: %{
```
Comment (bbrewer-papa-x) [inline apps/discovery_api/config/integration.exs:52]: Did you add these to the runtime.exs as well? If you can find a way to make this suck less for DRYness, please do it :)

File: apps/discovery_api/lib/discovery_api/search/dataset_index.ex [hunk 2]
```diff
@@ -222,8 +233,22 @@ defmodule DiscoveryApi.Search.DatasetIndex do
     |> Map.drop([:completeness])
     |> Enum.reject(fn {_k, v} -> is_nil(v) end)
     |> Map.new()
+    |> populate_org_facets()
```
Comment (bbrewer-papa-x) [inline apps/discovery_api/lib/discovery_api/search/dataset_index.ex:236]: Just makes cents

File: apps/discovery_api/lib/discovery_api_web/controllers/multiple_metadata_controller.ex [hunk 3]
```diff
@@ -101,6 +103,29 @@ defmodule DiscoveryApiWeb.MultipleMetadataController do
...
+    api_accessible = parse_api_accessible(params)
+
+    case validate_facets(facets) do
+      {:ok, filter_facets} ->
+        opts =
+          [
+            query: query,
+            api_accessible: api_accessible,
+            keywords: Map.get(filter_facets, :keywords),
+            org_title: Map.get(filter_facets, :organization, []) |> List.first()
```
Comment (bbrewer-papa-x) [inline apps/discovery_api/lib/discovery_api_web/controllers/multiple_metadata_controller.ex:118]: I hope someday that this `org_title` name will flow downstream into the UI (in vX.0 of the API) someday so we're no longer filtering on the hyper ambiguous `organization`

File: apps/discovery_api/config/integration.exs (hunk 1 above)
Comment (LtChae) [reply apps/discovery_api/config/integration.exs]: Have now

File: apps/discovery_api/lib/discovery_api_web/controllers/multiple_metadata_controller.ex (hunk 3 above)
Comment (LtChae) [reply apps/discovery_api/lib/discovery_api_web/controllers/multiple_metadata_controller.ex:118]: Might be a good enhancement once we get it wired up to the ui

**Pull request merged**